### PR TITLE
Changes

### PR DESCRIPTION
**Pull request by AlexOrd**

### CODE REVIEW

**Comment by IoanaGeorgia** [comment]
Ioana was here 👀

**Comment by hassanidris** [comment]
Check closing tags

**Comment by sameena83** [comment]
A bit broken

**Comment by ranaashraf2627** [comment]
ok

**Comment by MirinkaKotze** [comment]
Broken code - check tags

File: index.html [hunk 1]
```diff
@@ -11,7 +11,8 @@
 <body>
   <a href="/beetroot-projects/lesson-1/">HW Lesson 1</a>
   <a href="/beetroot-projects/lesson-2/">HW Lesson 2</a>
-  <a href="/beetroot-projects/lesson-3/">HW Lesson 3</a>
+  <a href="/beetroot-projects/lesson-3/">HW Lesson 3</a
```
Comment (RalenneB) [inline index.html]: everything is fine. :) almost :P

File: index.html [hunk 2]
```diff
@@ -11,7 +11,8 @@
 <body>
   <a href="/beetroot-projects/lesson-1/">HW Lesson 1</a>
   <a href="/beetroot-projects/lesson-2/">HW Lesson 2</a>
-  <a href="/beetroot-projects/lesson-3/">HW Lesson 3</a>
```
Comment (AlexOrd) [inline index.html:14]: Please fix it.

File: index.html (hunk 1 above)
Comment (hassanidris) [inline index.html]:
```suggestion
  <a href="/beetroot-projects/lesson-3/">HW Lesson 3</a>
```

File: index.html (hunk 2 above)
Comment (IoanaGeorgia) [inline index.html:14]: needs fixing urgently

**Review by MirinkaKotze — APPROVED** [review]

**Review by IoanaGeorgia — APPROVED** [review]
I approve 😸

**Review by sameena83 — APPROVED** [review]

**Review by MzfRnd — APPROVED** [review]

**Review by RalenneB — APPROVED** [review]

**Review by hassanidris — APPROVED** [review]

**Pull request merged**